src/Employer/Esfa.Recruit.Azure.Resources/azuredeploy.json

```diff
@@ -169,12 +177,19 @@
     "EmployerFinanceSqlDbConnectionString": {
       "type": "securestring",
       "defaultValue": ""
+    },
+    "ProviderCustomHostname": {
+      "type": "string"
+    },
+    "ProviderKeyvaultCertificateName": {
+      "type": "string"
     }
   },
   "variables": {
     "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
     "webJobAppServicePlanName": "[concat('das-', toLower(parameters('environmentName')),'-rcrt-webjob-asp')]",
     "EmployerRecruitAppServiceName": "[concat('das-', toLower(parameters('EnvironmentName')), '-rcrt-as')]",
+    "ProviderRecruitAppServiceName" :  "[concat('das-', toLower(parameters('EnvironmentName')), '-rcrtpr-as')]",
```

Comment:
Can we updated "rcrtpr-as" to "rcrt-pr-as" to align with the other names please?